core/src/main/kotlin/br/com/concrete/yosef/api/property/size/WidthCommand.kt

```diff
@@ -15,9 +18,26 @@ class WidthCommand : DynamicPropertyCommand {
          * This constant documents which name is associated with the property
          */
         const val WIDTH_TYPE = "width"
+        const val MATCH = "match"
```

Comment:
This is great, let's just see if it's compatible with iOS :apple: .

Reply:
I talked to an iOS engineer working on this and he said its fine, but I can hide this prop so its not misused now